charts/gitops-server/values.yaml

```diff
@@ -11,48 +11,58 @@ image:
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
-# Sets the WEAVE_GITOPS_AUTH_ENABLED environment variable to enable auth systems
+# -- Enable the login system.
 enableLogin: true
-# valid levels are 'debug', 'info', 'warn' and 'error'
+# -- What log level to output. Valid levels are 'debug', 'info', 'warn' and 'error'
 logLevel: info
 # Additional arguments to pass in
 # additionalArgs:
 # Any other environment variables:
 # envVars:
 
 serviceAccount:
-  # Specifies whether a service account should be created
+  # -- Specifies whether a service account should be created
   create: true
-  # Annotations to add to the service account
+  # -- Annotations to add to the service account
   annotations: {}
-  # The name of the service account to use.
+  # -- The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 rbac:
-  # Specifies whether the clusterRole & binding to the service account should be created
+  # -- Specifies whether the clusterRole & binding to the service account should be created
   create: true
-  # List of resources that the service account can impersonate. It is strongly
-  # recommended that you configure groups that can be used to populate this.
+  # -- If non-empty, this limits the resources that the service
+  # account can impersonate. It is strongly recommended that you
+  # configure groups that can be used to populate this.
   impersonationResourceNames: []
-  # Limit the type of principal that can be impersonated
+  # -- Limit the type of principal that can be impersonated
   impersonationResources: ["users", "groups"]
-  # Specify whether additional permissions should be granted to the service account
-  # in order to work with weave-gitops enterprise
+  # -- Specify whether additional permissions to view secrets should
```

Comment:
I think this is very wrong (I think I should have done a second pass on the docs when I added this). It should be something more like:
```
# -- Specify specific secrets the service account that can access, for example the
# enterprise entitlement.
```
we should probably rename `viewSecrets` to `viewSecretsResources` to make it clear it expects a list of resources and match naming with `impersonationResources`

Reply:
It matches `impersonationResourceNames`, so I called it `viewSecretResourceNames` instead.

Reply:
thank you